content/docs/ops/create-org.md

```diff
@@ -15,14 +15,14 @@ Non-sandbox organizations need to be created by hand. To do so:
 
 1. The [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1) should be filled out by a cloud.gov team member or the tenant.
 1. After requesting an org, a cloud.gov Business Unit person must verify the org request and check that all the necessary info is there. They automatically receive a notification (it goes to Zendesk), but if they haven't responded yet and you need a response soon, ask in #cloud-gov-business for a cloud.gov Business Unit person to take a look. They will verify the info and assign the Zendesk ticket to a person who can run the command (this may be you if you have permission).
-1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script. You will need the following information, which is all in the [form output](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=131031416) (consult the script itself for how to pass in the arguments):
-   * **Agency**: the requesting agency, column B of the form output.
-   * **Memory**: use the amount from column H of the form output.
-   * **System name**: the agency's system, column C of the form output.
-   * **Unique business ID**: agreement name, column F of form output.
+1. Clone [`cg-scripts`](https://github.com/18F/cg-scripts) to your computer, because you'll use the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script for this step. Use the command in column M of the [form output](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=131031416) for the appropriate row of the form - the spreadsheet should have automatically created the appropriate command using a formula. Using that script will automatically give the org the right name (column B + C).
 
 1. [Make the primary (technical) point of contact an `OrgManager`.]({{< relref "docs/apps/managing-teammates.md#give-roles-to-a-teammate" >}})
+    * If they don't have an account yet (for example if they don't have a GovCloud account yet), [invite them](https://account.fr.cloud.gov/invite) and ask them to accept the invite (and to let you know when they've accepted it). After they have an account, make them an OrgManager.
     * They can grant themselves and others additional access using those same instructions.
+    
+1. Remove yourself from any roles in the org that you automatically received.
```

Comment:
Oh weird, I manually removed Roger after he created an org today - it might not be working right. Next time we can look more closely at what happened so we can figure out where the problem is.